src/phoenix/server/app.py

```diff
@@ -103,6 +104,8 @@ class AppConfig(NamedTuple):
     min_dist: float
     n_neighbors: int
     n_samples: int
+    is_development: bool
+    manifest_json_path: Path
```

Comment:
manifest is kinda a generic name - I'd either document it or call it the app_manifest or web_manifest?

Reply:
good call